src/containers/ProfilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
Your container have much more css html designing stuff dont do it like this. Consider refactoring your components again, So that container page will only use the custom react components a few divs.